js_basics_functions/task3.js

```diff
@@ -0,0 +1,10 @@
+const recursion = num =>  {
```

Comment:
function name does not represent what function actually does. Use more informative name like `countDown`